testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/ResultFragment.java

```diff
@@ -43,9 +43,16 @@
 public class ResultFragment extends Fragment {
 
     public static final String CORRELATION_ID = "correlation_id";
+    public static final String ACCOUNT_TENANT_ID = "account_tenant_id" ;
```

Comment:
We probably should distinguish between home tenant and guest tenants and their respective identifiers.  I think the account object has these different IDs are part of tenant profiles.